test/unit/uk/gov/hmrc/openidconnect/userinfo/services/UserInfoTransformerSpec.scala

```diff
@@ -193,5 +201,18 @@ class UserInfoTransformerSpec extends UnitSpec with MockitoSugar {
       val userInfoMissingPostCode = userInfo.copy(address = Some(userAddress.copy(formatted = "1 Station Road\nTown Centre\nLondon\nEngland\nUK\nUnited Kingdom\nGB", postal_code = None)), hmrc_enrolments = None, email = None)
       result shouldBe userInfoMissingPostCode
     }
+
+    "not return country code when feature flag is off" in new Setup {
+
+      FeatureSwitch.disable(UserInfoFeatureSwitches.countryCode)
+
+      val scopes = Set("address", "profile", "openid:gov-uk-identifiers", "openid:government_gateway")
+
+      val desUserMissingPostCode = desUserInfo.copy(address = desAddress.copy(postcode = None))
```

Comment:
Not really and it would be slightly clearer without I guess, but because I have added a single test for 'feature flag' on and it is based on the existing test which has a postcode missing, it is easier to go with it this way.
Ultimately, once the when the feature flag goes so will this test.